qemu/tests/pci_bridge_hotplug_unplug.py

```diff
@@ -0,0 +1,140 @@
+import logging
+import time
+
+from virttest import error_context
+from virttest import utils_test
+
+
+def disk_hotplug(test, vm, params, image, drive_format, parent_bus):
+    def _get_index(index):
+        while vm.index_in_use.get(str(index)):
+            index += 1
+        return index
+
+    def get_index(vm, params, image_params):
```

Comment:
The image index is not indispensable, so I remove get_index function here. Thanks!